client/components/LevelThree.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const LevelThree = (props) => {
+  const {
+    name, address, city, state, zip, change, date, time, book,
+    first, last, email, phone, count, back,
+  } = props;
+
+  const addressString = `${city} ${state}, ${zip}`;
+
+  let isValid = false;
+  if (first && last && email && phone) {
+    isValid = true;
+  }
+
+  const notValid = (event) => {
+    event.preventDefault();
+    alert('Must complete the required fields.');
+  };
+
+  return (
+    <form>
+      <div>
+        <div>
+          First Name*
+          <input onChange={change} type="text" name="firstName" value={first} />
+          Last Name*
+          <input onChange={change} tyep="text" name="lastName" value={last} />
+        </div>
+        <div>
+          Email*
+          <input onChange={change} name="email" value={email} />
+          Mobile Number*
+          <input onChange={change} name="phone" value={phone} />
+        </div>
+        <div>
+          * = required field
+        </div>
+      </div>
+      <br />
+      <div>
+        <div>
+          {date}
+          {' at '}
+          {time}
+        </div>
+        <div>
+          {name}
+        </div>
+        <div>
+          {address}
+          <br />
+          {addressString}
+        </div>
+        <div>
+          Guests: &nbsp;
```

Comment:
I struggled with this a bit on my project.  Apparently the nbsp can cause errors in tests.  I ended up going with the string template literal {`Guests:  `}.  For me, at least, the backticks preserved my blank spaces even when regular quotes didn't.

Reply:
Good call! Works great!